src/test/java/com/redhat/devtools/lsp4ij/features/foldingRange/TypeScriptFoldingRangeTest.java

```diff
@@ -75,4 +76,49 @@ export class Demo {<start1>
                         """
         );
     }
+
+    // Collapse-by-default tests
```

Comment:
And I added more tests to the existing TypeScript test suite specifically for its support for the `imports` folding range kind, and as above, I confirm that regions are collapsed-by-default as appropriate based on the respective IDE configuration setting.